source/includes/options-mongosqld.yaml

```diff
@@ -828,4 +828,26 @@ description: |
      +------+------+--------+-------+
 
 optional: true
+---
+program: mongosqld
+name: setParameter
+directive: option
+args: <polymorphic_type_conversion_mode | type_conversion_mode>
+description: |
+
+  .. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Corresponds to
+
+   * - ``polymorphic_type_conversion_mode``
+     - string
+     - Determines how :binary:`~bin.mongosqld` evaluates document fields that are specified with multiple data types.
+
+   * - ``type_conversion_mode``
+     - string
+     - Specifies the semantics that :binary:`~bin.mongosqld` uses for type conversions, such as the CAST function.
```

Comment:
Add a supplemental link to the system variables page that describes this behavior a bit better:
https://docs.mongodb.com/bi-connector/current/reference/system-variables/

```
For more information on type conversion modes, see :ref:`system-environment-variables`.
```